secure-app-model/keyvault/cpvsample/pom.xml

```diff
@@ -22,9 +22,14 @@
 			<version>1.18.0</version>
 		</dependency>
 		<dependency>
-			<groupId>com.microsoft.azure</groupId>
-			<artifactId>azure-keyvault</artifactId>
-			<version>1.2.2</version>
+			<groupId>com.azure</groupId>
```

Comment:
The latest packages are:
azure-identity: 1.1.2 and azure-keyvault: 4.2.1

https://azure.github.io/azure-sdk/#java